compiler/ffi/ffi_parser.cpp

```diff
@@ -124,5 +135,15 @@ std::pair<const FFIType*, FFIParseError> ffi_parse_type(const std::string &type_
   } catch (ffi::ParsingDriver::ParseError &e) {
     set_error(err, src, e);
   }
+
+  // wrap the result_type in array types
+//  while (array_dims > 0) {
+//    auto *array_type = new FFIType{FFITypeKind::Array};
+//    array_type->members = {result_type};
+//    array_type->num = -1;
+//    result_type = array_type;
+//    array_dims--;
+//  }
```

Comment:
Here (and above) lines are commented out, is this intentional?

Reply:
Good catch.
It's a leftover from the time before I fixed the C parser grammar file.
Now it can be parsed without this kludge.

I removed the comment.